backend/app/store/service/service_test.go

```diff
@@ -738,12 +752,12 @@ func TestService_SetMetas(t *testing.T) {
 	// two comments for https://radio-t.com
 	b := DataStore{Engine: prepStoreEngine(t), EditDuration: 100 * time.Millisecond,
 		AdminStore: admin.NewStaticKeyStore("secret 123")}
-	umetas := []UserMetaData{}
-	pmetas := []PostMetaData{}
+	var umetas []UserMetaData
```

Comment:
minor: I guess your IDE suggested this change, because i see no real tecnnical reason to do it. This is style question and I'd suggest not to change tt in fragments you modify unless you have a good reason

Reply:
You are absolutely right, I've cleaned up all these changes in last few commits but missed this one.